string goal_turtle_name
---
int32[] distance
---
int32[] partial_distance



